reference/debugging.md

```diff
@@ -0,0 +1,150 @@
+# Debugging
+
+If you’re experiencing issues with the Ambassador Edge Stack and cannot diagnose the issue through the "Diagnostics" tab from the [Edge Policy Console](../../about/edge-policy-console), this document covers various approaches and advanced use cases for debugging Ambassador issues.
+
+We assume that you already have a running Ambassador installation in the following sections.
+
+## Check Ambassador Status
+
+First, check to see if the [Diagnostics](../diagnostics) service is reachable. If it is successful, try to diagnose your original issue with the Diagnostics Console.
+
+**If it is not successful, complete the following to see if Ambassador is running:**
+
+1. Get a list of Pods in the `ambassador` namespace with `kubectl get pods -n ambassador`.
+
+    The terminal should print something similar to the following:
+
+    ```console
+    $ kubectl get pods -n ambassador
+    NAME                         READY     STATUS    RESTARTS   AGE
+    ambassador-85c4cf67b-4pfj2   1/1       Running   0          1m
+    ambassador-85c4cf67b-fqp9g   1/1       Running   0          1m
+    ambassador-85c4cf67b-vg6p5   1/1       Running   0          1m
+    ```
+
+2. Choose a Pod that you want to examine in the Diagnostics Console with: `kubectl port-forward -n ambassador <ambassador-pod-name> 8877`.
```

Comment:
Delete step 2 -- it's just a way to get access to the diagnostics UI from outside the cluster, and you don't use it anywhere below that I see.

Reply:
removed